lib/build_runner.zig

```diff
@@ -240,11 +259,13 @@ fn usage(builder: *std.Build, already_ran_build: bool, out_stream: anytype) !voi
     }
 
     try out_stream.print(
+        \\
         \\Usage: {s} build [steps] [options]
         \\
         \\Steps:
         \\
     , .{builder.zig_exe});
+    try steps(builder, true, out_stream);
```

Comment:
This looks like it will repeat the steps twice with the loop below.

Reply:
Not sure how I missed that... It's fixed now.